frontend/.env.template

```diff
@@ -0,0 +1,2 @@
+GOOGLE_MAPS_API_KEY_IOS=YOUR_API_KEY_HERE
```

Comment:
This is nitpicky, but name this file "sample.env" to be consistent with the backend. Otherwise, looks good.